packages/react/src/components/DataTable/TableToolbarSearch.tsx

```diff
@@ -159,9 +142,12 @@ const TableToolbarSearch = ({
   ...rest
 }: TableToolbarSearchProps) => {
   const { current: controlled } = useRef(expandedProp !== undefined);
-  const [expandedState, setExpandedState] = useState<
-    string | boolean | undefined
-  >(defaultExpanded || defaultValue);
+
+  // TODO: Why does defaultValue affect the initial expandedState?  This is from #5089.
+  const [expandedState, setExpandedState] = useState<boolean>(
+    Boolean(defaultExpanded || defaultValue)
+  );
```

Comment:
@wkeese Would this be because if there is a `defaultValue` the input should be expanded by default? That's the primary reason I can think of.

Reply:
I guess you're right, actually I'm unclear what "expanded" means for a search toolbar...

In any case I removed the TODO comment.